klasses/api_test.py

```diff
@@ -38,6 +39,27 @@ def setup_test_data():
     order = OrderFactory.create(user=user, status=Order.FULFILLED)
     LineFactory.create(order=order, klass_id=klass_paid.klass_id, price=627.34)
 
+    mocker.patch(
+        'klasses.bootcamp_admissions_client.BootcampAdmissionClient._get_admissions',
+        autospec=True,
+        return_value={
+            "user": user.email,
+            "bootcamps": [
+                {
+                    "bootcamp_id": None,  # NOTE:this is the ID on the remote web service (we do not need it)
+                    "bootcamp_title": bootcamp.title,
+                    "klasses":  [
+                        {
+                            "klass_id": klass.klass_id,  # NOTE: this is the ID on the remote web service
+                            "klass_name": klass.title,
+                            "is_user_eligible_to_pay": True
+                        } for klass in bootcamp.klass_set.order_by('id')
+                    ]
+                } for bootcamp in Bootcamp.objects.all().order_by('id')
+            ]
+        }
+    )
```

Comment:
here's another simple pattern to achieve this with an actual fixture: http://stackoverflow.com/questions/33508060/create-and-import-helper-functions-in-tests-without-creating-packages-in-test-di/42156088#42156088